string.js

```diff
@@ -0,0 +1,16 @@
+
+// 1- Remove the commas from the string and replace them with spaces.
+
+const myString = "hello,this,is,a,difficult,to,read,sentence";
+console.log (myString.length);
+
+let newStr = myString.replace(/,/gi, " ");
+console.log(newStr);
+
```

Comment:
Correct. Just that 'i' in /,/gi is not needed (there is no 'caseuality' in a comma)